### DISCUSSION TITLE
How much bandwidth does Slinger server uses?

### DISCUSSION THREAD

**Discussion by mike-uk** [GerryDazoo/Slinger#585] [Q&A] [answered]
I just wonder does Slinger server use much bandwidth? Is it mainly a pointer to redirect the address details between the Slingbox and the player location?

Thanks

**Comment by bradut** [comment]
**Bandwidth**:
In my case, streaming Standard Definition uses 2.8 ... 3 Mbps / Player. When running two Slinger Players in paralel it spiked to 8Mbps.

**Reply by mike-uk** [reply]
But is that between the sling box and sling player only or the data have to go through the server between the 2?

**Comment by GerryDazoo** [comment, marked as answer]
The server does not add any overhead to the data stream from the slingbox. Once the stream is set up it's basically a packet in/packet out engine. So the bandwidth used is whatever the sling is sending.

**Reply by mike-uk** [reply]
I mean if the server is at location A, and sling box at B and player at C. Are the streaming happening B and C only? Does location A use much data?

**Reply by GerryDazoo** [reply]
B -> A -> C

**Reply by mike-uk** [reply]
So weak connection at A can affect the play back at C even when B is has high bandwidth connection?

**Reply by GerryDazoo** [reply]
Yes

**Comment by robosmith** [comment]
> The server does not add any overhead to the data stream from the slingbox. Once the stream is set up it's basically a packet in/packet out engine. So the bandwidth used is whatever the sling is sending.

Didn't you say that Slinger has to process additional (decryption?) data for the Pro-HD?
That doesn't increase the data stream overhead?

**Reply by GerryDazoo** [reply]
No extra bandwidth required just some CPU. the decryption takes 128 bit chunks of data and turns it into 128 bits of usable data.
